jvm-libs/linea/blob-decompressor/src/test/kotlin/net/consensys/linea/blob/RLPHelper.kt

```diff
@@ -0,0 +1,26 @@
+package net.consensys.linea.blob
+
+import org.apache.tuweni.bytes.Bytes
+import org.hyperledger.besu.ethereum.rlp.RLP
+
+internal fun rlpEncode(list: List<ByteArray>): ByteArray {
+  return RLP.encode { rlpWriter ->
+    rlpWriter.startList()
+    list.forEach { bytes ->
+      rlpWriter.writeBytes(Bytes.wrap(bytes))
+    }
+    rlpWriter.endList()
+  }.toArray()
+}
```

Comment:
@jpnovais A late question, not a blocker to this PR, why not put them in `ByteArrayExtension`?

Reply:
Because generic/extensions/XYZ, must be vanila packages that do not bring extra dependencies. To do that, I would need to add Besu libraries to `generic/extensions/kotlin`.